### PR TITLE
rose doc: update [poll] delays to ISO8601

### PR DESCRIPTION
**Pull request by kaday**
Update to documentation to reflect that the code supports ISO8601 date-time format

### CODE REVIEW

**Comment by kaday** [comment]
@arjclark please review as discussed

File: doc/rose-rug-task-run.html [hunk 1]
```diff
@@ -338,22 +338,24 @@ <h3 id="rose-task-run.poll">Poll for Prerequisites</h3>
     with delays between them. If the prerequisites are still not met after the
     number of delays, <code>rose task-run</code> will fail with a time out. The
     delays list is a comma-separated list. The syntax looks like
-    <var>[R*]T[U]</var>, where <var>U</var> is a unit (<kbd>s</kbd> for seconds
-    (default), <kbd>m</kbd> for minutes and <kbd>h</kbd> for hours),
-    <var>T</var> is the number of units, and <var>R</var> is the number of
-    repeats. E.g.:</p>
+    <var>[R*]P[D]</var>, where <var>D</var> is a duration in time (for example,
+    <kbd>1D</kbd> for 1 day, <kbd>1DT6H</kbd> for 1 day and 6 hours) with units
+    (<kbd>S</kbd> for seconds (default), <kbd>M</kbd> for minutes and
```
Comment (arjclark) [inline doc/rose-rug-task-run.html]: <N>M for N months, T<N>M for N minutes.

File: doc/rose-rug-task-run.html [hunk 2]
```diff
@@ -338,22 +338,24 @@ <h3 id="rose-task-run.poll">Poll for Prerequisites</h3>
...
+    <kbd>H</kbd> for hours), <var>P</var> is the ISO8601 date-time format, and
+    <var>R</var> is the number of repeats. E.g.:</p>
```
Comment (arjclark) [inline doc/rose-rug-task-run.html]: Perhaps put R first in the listing as its the first thing they might encounter?

**Comment by kaday** [comment]
@arjclark tried again, a slightly different tack.

**Comment by arjclark** [comment]
Looks ok to me.

**Pull request merged**